#!/usr/bin/env python
PACKAGE = "sub_trajectory"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("depth_vel_P", double_t, 0, "Proportional gain for depth velocity zeroing", 1, 0)
gen.add("depth_pos_P", double_t, 0, "Proportional gain for depth position holding", 1, 0)

gen.add("orientation_vel_P", double_t, 0, "Proportional gain for orientation velocity zeroing", 1, 0)
gen.add("orientation_pos_P", double_t, 0, "Proportional gain for orientation position holding", 0.1, 0)


exit(gen.generate(PACKAGE, "active_stability", "Stability"))